web/package-lock.json

```diff
@@ -2390,21 +2390,6 @@
       "integrity": "sha512-Kvp459HrV2FEJ1CAsi1Ku+MY3kasH19TFykTz2xWmMeq6bk2NU3XXvfJ+Q61m0xktWwt+1HSYf3JZsTms3aRJg==",
       "peer": true
     },
-    "node_modules/@jest/core/node_modules/fsevents": {
```

Comment:
let's revert these package-lock changes unless something needs to be updated/installed